from-kokoro.sh

```diff
@@ -146,7 +146,7 @@ lint_clang_format() {
     # Install clang-format
     echo "Downloading clang-format..."
     curl -Ls "https://github.com/material-foundation/clang-format/releases/download/$CLANG_FORMAT_TAG/clang-format" -o "clang-format"
-    if openssl sha -sha256 "clang-format" | grep -q "$CLANG_FORMAT_SHA"; then
+    if openssl sha256 -sha256 "clang-format" | grep -q "$CLANG_FORMAT_SHA"; then
```

Comment:
Did you confirm that this a backward-compatible change? If not, I'd expect this to be a major version bump.

Reply:
My understanding is that the `openssl sha256` command is the correct invocation, and that `openssl sha` has gone away but I can't find any announcements confirming this. With this information I believe this constitutes a patch fix because it addresses broken functionality in newer CI machines.